client/components/includes/header-side-bar.js

```diff
@@ -1,121 +1,63 @@
+              <div className="row style={{backgroundColor: '#25758c'}}">
+                <span className="card-title"><h4><i className="material-icons">library_books</i> HelloBooks</h4></span>
+                <li className="divider"></li>
+                <p></p>
+                <img style={{ borderRadius: 50, border: '2px solid black' }}
+                  width="100px"
+                  height="100px"
+                  src="https://images.vexels.com/media/users/3/130527/isolated/preview/845f79841ea58765d623a68bf434d5ed-girl-cartoon-head-character-by-vexels.png"
+                  alt="HelloBooks" /><br />
+                <i className="material-icons">account_circle</i> <b>{this.props.fullName}</b>
+              </div><br />
+            </div>
+            <li className="divider"></li>
+            <li id="menu-list"><a href='rented-books'>Rent History <i className="material-icons">chevron_right</i></a></li>
+            <li id="menu-list"><a href="/dashboard">Rent a Book <i className="material-icons">chevron_right</i></a></li>
+            <li id="menu-list"><a href="#!">Rented Books <i className="material-icons">chevron_right</i></a></li>
+            <li id="menu-list"><a href="#!">Rent History <i className="material-icons">chevron_right</i></a></li>
           </ul>
-        </div>
-      </li>
-    </ul>
-  </ul>
-
-  <header>
-    <ul className="dropdown-content" id="user_dropdown">
-      <li><a className="indigo-text" href="#!">Profile</a></li>
-      <li><a className="indigo-text" onClick={this.props.onClick} href="">Logout</a></li>
-    </ul>
-
-    <nav className="header-side" role="navigation">
-      <div className="nav-wrapper">
-        <a data-activates="slide-out" className="button-collapse show-on- text-large" href="#!">HelloBooks</a>
-
-        <ul className="right hide-on-med-and-down">
-          <li>
-            <a className='right dropdown-button' href='' data-activates='user_dropdown'><i className=' material-icons'>account_circle</i></a>
-          </li>
-        </ul>
+        </div></div>
+      <div id="content">
+        <a href="#" data-activates="slide-out" className="button-collapse hide-on-large-only"><i style={{ color: '#fff', fontSize: 40 }} className="material-icons">menu</i></a>
 
-        <a href="#" data-activates="slide-out" className="button-collapse"><i className="mdi-navigation-menu"></i></a>
       </div>
-    </nav>
-
-    <nav>
-      <div className="nav-wrapper header-side darken-2">
-        <a style={{marginLeft: 20}} className="breadcrumb" href="#!">Admin</a>
-        <a className="breadcrumb" href="#!">Index</a>
-
-        <div style={{marginRight: 20}} id="timestamp" className="right"></div>
-      </div>
-    </nav>
-  </header>
-
-            </div>
-        )
-    }
+    </div>)
+  }
 }
 
 HeaderSideBar.PropTypes = {
-  fullname: PropTypes.string.isRequired,
+  fullName: PropTypes.string.isRequired,
   logout: PropTypes.func.isRequired
 }
```

Comment:
Go back to all your files and make sure to add a newline to the end of all files.